app/views/wpcc/shared/_your_details.html.erb

```diff
@@ -0,0 +1,14 @@
+<section class="section section--details details">
+  <h2 class="section__heading details__heading">1. <%= t('wpcc.details.title') %>
+  <span class="section__heading-summary">
+    (<%= session[:age] %> years,
+      <%= session[:gender] %>,
+      £<%= session[:salary] %> 
+      <%= session[:salary_frequency] %>,
+      <%= session[:contribution_preference] %> Contribution)
```

Comment:
`<%= session[:contribution_preference] %> Contribution)` the word contribution needs to be translated into Welsh. This can ear-marked as work for future improvement.